gossip-bin/src/ui/mod.rs

```diff
@@ -1367,21 +1372,48 @@ impl eframe::App for GossipUi {
         // Side panel
         self.side_panel(ctx);
 
+        let has_warning = {
+            #[cfg(feature = "video-ffmpeg")]
+            {
+                self.audio_device.is_none()
+            }
+            #[cfg(not(feature = "video-ffmpeg"))]
+            {
+                false
+            }
+        };
+
         egui::TopBottomPanel::top("top-area")
             .frame(
                 egui::Frame::side_top_panel(&self.theme.get_style()).inner_margin(egui::Margin {
-                    left: 20.0,
-                    right: 15.0,
-                    top: 10.0,
-                    bottom: 10.0,
+                    left: 30.0,
+                    right: 30.0,
+                    top: 20.0,
+                    bottom: 20.0,
                 }),
             )
             .resizable(true)
             .show_animated(
                 ctx,
-                self.show_post_area_fn() && read_setting!(posting_area_at_top),
+                (self.show_post_area_fn() && read_setting!(posting_area_at_top)) || has_warning,
```

Comment:
The OR with `has_warning` here will make sure that the top sidebar is open when the warning is active.